src/classes/PHPUnit/Actor.php

```diff
@@ -697,6 +697,20 @@ public function login( string $username = null, string $password = 'password' )
 			// Do nothing
 		}
 
+		// Fix Update WordPress Database prompt
+		try {
+			sleep(1);
```

Comment:
I think this can be done in a better way. This would make every login at least 1 second slower.

Reply:
@tlovett1 any recommendations on alternate approaches that will solve the root issue?

Reply:
@tlovett1 Ah I see, the fix is not reliable as well. Closing it for now until I find better way to do this. Right now, I solve this issue at project level by running `wp core update-db` in `setUp` method.